### ISSUE TITLE
Zooming in Chrome causes truncation to empty content at certain levels

### ISSUE THREAD

**Issue by jordanmkoncz** [BeSite/jQuery.dotdotdot#55]
In Chrome (v39.0.2171.95), zooming to 75% or 25% causes content that is being truncated by dotdotdot to be truncated to nothing (removes **all** content from element). This doesn't seem to happen at any other zoom levels, or in any of the other browsers I tested (Firefox, Internet Explorer 9).

Here's my dotdotdot config:
```
$('.js-truncate').dotdotdot({
	watch: 'window',
	after: 'a.read-more'
});
```

**Comment by jordanmkoncz** [comment]
Okay so after some further testing, I think I've found what causes this to occur.

The text content that this was happening for was a `div` that I wanted dotdotdot to truncate such that it's always a single line of text. In Chrome, the height of this `div` is 27px, so to achieve what I wanted, I had set `height: 27px` on the `div` that contains this text content.

Now when I zoom to 75% in Chrome, Chrome believes the height of this same `div` is actually 28px rather than 27px. Then, the dotdotdot plugin truncates the content to nothing, presumably because it thinks the content is too big to fit in the `div` as a single line due to the `height` property of the `div` being manually set to 27px, but the height of the content actually being 28px at the 75% zoom level.
